File: app/src/main/java/com/poc/paphoscafe/service/model/NotificationBody.java
package com.poc.paphoscafe.service.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class NotificationBody {
    @SerializedName("badge")
    @Expose
    private int badge;

    @SerializedName("sound")
    @Expose
    private String sound;

    @SerializedName("body")
    @Expose
    private String body;

    public int getBadge() {
        return badge;
    }

    public void setBadge(int badge) {
        this.badge = badge;
    }

    public String getSound() {
        return sound;
    }

    public void setSound(String sound) {
        this.sound = sound;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }
}

File: app/src/main/java/com/poc/paphoscafe/service/model/NotificationBody.kt
package com.middleware.poc.service.model

import com.google.gson.annotations.Expose

import com.google.gson.annotations.SerializedName

class NotificationBody {
    @SerializedName("badge")
    @Expose
    private var badge: Int? = null

    @SerializedName("sound")
    @Expose
    private var sound: String? = null

    @SerializedName("body")
    @Expose
    private var body: String? = null

    fun getBadge(): Int? {
        return badge
    }

    fun setBadge(badge: Int?) {
        this.badge = badge
    }

    fun getSound(): String? {
        return sound
    }

    fun setSound(sound: String?) {
        this.sound = sound
    }

    fun getBody(): String? {
        return body
    }

    fun setBody(body: String?) {
        this.body = body
    }

    override fun toString(): String {
        return "{notification: {body: $body}}"
    }
}